gen/dcos-config.yaml

```diff
@@ -447,8 +447,6 @@ package:
       {
         "PATH": "/usr/bin:/bin",
         "SHELL": "/usr/bin/bash",
-        "LD_LIBRARY_PATH": "/opt/mesosphere/lib",
```

Comment:
I understand how long we've been dragging this out far too long, but I don't think breaking all our frameworks is going to result in a solution either. The real solution will be to provide them with the tools/APIs (libmesos bundle and v1 HTTP API) so that they can move off of host libmesos with ease and confidence. And those are the efforts that @karya0 and I have been involved in recently.